components/chunking/chunk-db-write-impl/src/main/kotlin/net/corda/chunking/db/impl/persistence/database/DatabaseCpiPersistence.kt

```diff
@@ -112,26 +113,44 @@ class DatabaseCpiPersistence(private val entityManagerFactory: EntityManagerFact
         }
     }
 
+    /**
+     * Update the changelogs in the db for force upload
```

Comment:
I do not think the call to this method is happening **only** during force CPI upload, therefore: 

```suggestion
     * Update the changelogs in the db during CPI upload
```